trunk/source/CACSL2BoogieTranslator/src/de/uni_freiburg/informatik/ultimate/cdt/translation/implementation/base/ACSLHandler.java

```diff
@@ -173,6 +175,8 @@ private enum SPEC_TYPE {
 	private final CHandler mCHandler;
 	private final CExpressionTranslator mCExpressionTranslator;
 
+	private final ScopedHashMap<String, LRValue> mQuantifiedVariables = new ScopedHashMap<>();

```

Comment:
Maybe `mBoundVariables`, so it also makes sense if support for `let` or `lambda` is added.